resolwe/flow/managers/workload_connectors/kubernetes.py

```diff
@@ -493,9 +493,18 @@ def start(self, data: Data):
         requests["cpu"] = limits.pop("cores")
         limits["cpu"] = requests["cpu"] + 1
 
-        requests["memory"] = limits["memory"] * (10 ** 6)
-        limits["memory"] += KUBERNETES_MEMORY_HARD_LIMIT_BUFFER
+        # The memory in the database is stored in megabytes but the kubertenes
+        # requires memory in bytes.
+        # We request 10% less memory than stored in the database asd set limit
+        # at 10% more plus KUBERNETES_MEMORY_HARD_LIMIT_BUFFER. The processes
+        # usually require 16GB, 32GB... and since the node usualy has 64GB of
+        # memory and some of it is consumed by the system processes only one
+        # process process that requires 32GB can run on a node instead of 2.
+
+        requests["memory"] = 0.8 * limits["memory"]
+        limits["memory"] = 1.1 * limits["memory"] + KUBERNETES_MEMORY_HARD_LIMIT_BUFFER
         limits["memory"] *= 10 ** 6
+        requests["memory"] *= 10 ** 6
```

Comment:
Is it ok to use `10 ** 6` here or would it be better to change it to `2 ** 20`?

Reply:
We have to decide. I believe (2 ** 20) in indeed a better choice.
But how about storage? Should we stick with the same units?